lib/foreman_tasks/dynflow/configuration.rb

```diff
@@ -59,6 +59,8 @@ def run_on_init_hooks(world)
 
     def initialize_world(world_class = ::Dynflow::World)
       world_class.new(world_config)
+    rescue ArgumentError
```

Comment:
In that case, the dummy world should start accepting the additional argument. Alternatively, we could just put expectation about invoking the `initialize_world` method, and make assertions around the `world_config`.